server/preprocessing/other-scripts/vis_layout.R

```diff
@@ -17,7 +17,8 @@ debug = FALSE
 vis_layout <- function(text, metadata, max_clusters=15, maxit=500, mindim=2, maxdim=2, lang="english") {
   stops <- stopwords(lang)
   if (lang=="english"){
-    additional_stops <- scan("../../resources/english.stop", what="", sep="\n")
+    # relative path is problematic as file is called from different sources
```

Comment:
I also had the problem with single-word labels. Could resolve it with @chreman in a call...

I assume you recently updated your R-packages. Including the tm-package... We realized that tm 0.7-* breaks the labeling without throwing any errors/warnings. So downgrading to 0.6-* should resolve that problem. 
I updated the server requirements: 1e5f933

Agree with resolving the path issue before merging back into master!